scarb/src/ops/fmt.rs

```diff
@@ -12,9 +13,24 @@ use crate::core::workspace::Workspace;
 use crate::core::PackageId;
 use crate::internal::serdex::toml_merge;
 
+#[derive(Debug, Clone, ValueEnum)]
+pub enum EmitTarget {
+    Stdout,
+}
+
+/// Format option to display the output formatted file
+/// Example: scarb fmt --emit stdout
+#[derive(Debug, Default, Clone)]
+pub enum FmtAction {
+    #[default]
+    Fix,
+    Check,
+    Emit(EmitTarget),
+}
+
 #[derive(Debug)]
 pub struct FmtOptions {
-    pub check: bool,
+    pub action: FmtAction,
```

Comment:
I think this is cleaner with regards to business logic now that there is a third option (fix, check or emit)

Reply:
👍